client/src/components/StandardFAQ/faq_utils.js

```diff
@@ -0,0 +1,19 @@
+import _ from "lodash";
+import React from "react";
+
+import { create_text_maker_component } from "src/components/misc_util_components";
+
+import { Gov } from "src/models/organizational_entities";
+
+export const faq_content_maker = (
+  text_bundles,
+  q_a_base_keys,
+  subject = Gov
+) => {
+  const { TM } = create_text_maker_component(text_bundles);
+
+  return _.map(q_a_base_keys, (base_text_key) => [
+    <TM key={"q"} k={base_text_key + "_q"} args={{ subject }} />,
+    <TM key={"a"} k={base_text_key + "_a"} args={{ subject }} />,
+  ]);
```

Comment:
I'd kind of prefer consumers of the FAQ component to explicitly declare all their question answer pairs. For one, there's #1128, for another it will make the text args more flexible/clearer. Not all FAQs need a subject arg, and what if a future item needs args other than subjects, or if a given FAQ drawer needs to pass different subjects to different Q/A pairs? Made sense that all items would always be related to a given subject when the FAQs were just an infographic thing, but arbitrary routes outside the infographics don't have the same single-subject scope.

BUT, I'm fine with dropping this pattern being a follow up PR or issue, just to move this PR along (and because I'll admit I gave confusing feedback in my last review when I threw out three different solutions to this problem, haha, sorry) .